lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -20,113 +53,187 @@
             </adf-card-view>
         </mat-expansion-panel>
         <ng-container *ngIf="displayTags">
-            <mat-expansion-panel *ngIf="!editable">
+            <mat-expansion-panel 
+                            (opened)="handleTagsPanelState(true)"
+                            (closed)="handleTagsPanelState(false)"
+                            hideToggle
+                            [expanded]="tagsPanelState">
                 <mat-expansion-panel-header>
-                    <mat-panel-title>{{ 'METADATA.BASIC.TAGS' | translate }}</mat-panel-title>
+                    <div class="adf-metadata-properties-panel-content">
```

Comment:
This code is repetitive, please extract it to a separate reusable component to have single place with the mat-icon

```html
<adf-metadata-properties-panel-content [title]="'METADATA.BASIC.TAGS'>
```

And the whole div should be inside that component and reused for all the panels you use in this template